### PR TITLE
Dynamic properties for all entities

### PR DESCRIPTION
**Pull request by zCarot**
Quick fix for #77 and #78 for production environment

### CODE REVIEW

**Comment by toin0u** [comment]
@zCarot Thank you for your PR. I'll look at it asap as we need to make the lib less sensitive to API changes happened:
- https://github.com/toin0u/DigitalOceanV2/issues/42
- https://github.com/toin0u/DigitalOceanV2/issues/57
- https://github.com/toin0u/DigitalOceanV2/issues/77

/cc @yassirh @liverbool

**Comment by liverbool** [comment]
@toin0u Looks good for me!

File: src/Entity/AbstractEntity.php [hunk 1]
```diff
@@ -34,10 +36,15 @@ public function __construct($parameters)
...
     {
         if (!property_exists($this, $property)) {
+            if (array_key_exists($property, $this->_unknownProperties)) {
+                 return $this->_unknownProperties[$property];
+            }
             throw new \InvalidArgumentException(sprintf(
                 'Property "%s::%s" does not exist.', get_class($this), $property)
             );
         }
+
```
Comment (GrahamCampbell) [inline src/Entity/AbstractEntity.php]: These extra lines should be removed.

File: src/Entity/AbstractEntity.php [hunk 2]
```diff
@@ -16,6 +16,8 @@
  */
 abstract class AbstractEntity
 {
+    protected $_unknownProperties = array();
```
Comment (GrahamCampbell) [inline src/Entity/AbstractEntity.php]: Why the underscore? We can assume that digital ocean is never going to conflict with this anyway.

**Comment by toin0u** [comment]
@zCarot Thank you!

**Pull request merged**